build/flaws.js

```diff
@@ -336,10 +337,9 @@ function injectPreTagFlaws(level, doc, $, rawContent) {
     const flaw = { explanation, id, fixable, html, suggestion, type };
     if (fixable) {
       // Only if it's fixable, is the `html` perfectly findable in the raw content.
-      for (const { line, column } of findMatchesInText(html, rawContent)) {
-        flaw.line = line;
-        flaw.column = column;
-      }
+      const { line, column } = findFirstMatchInText(html, rawContent);
```

Comment:
This is a strange change. Are you certain it's right? 
This change assumes that it will always find a match. Before, it was "optional". 
I'm reading the `findFirstMatchInText` function and I can't immediately see how it would work if the needle is not in the haystack. 
If you certainly only wanted the *first* match of `line` and `column` you should have refactored the existing code to:
```diff
for (const { line, column } of findMatchesInText(html, rawContent)) {
  flaw.line = line;
  flaw.column = column;
  // Only ever interested in the first match, so we can bail now.
  break;
}
```
That way we don't have two implementations of finding needles in haystacks. One that's able to regex the needle and specifically look within HTML attributes. And one that only uses a plan String.indexOf.